types/grasp/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
```

Comment:
Also `"strictFunctionTypes": true`

Reply:
Ah it’s in there but I added it at the bottom - should I move that up just for consistency? (dts-gen didn’t add it, but it gave me the rest)

Reply:
Just copy from `abs`. And maybe file an issue on `dts-gen`.

Reply:
Ok, copied!

Reply:
Also https://github.com/Microsoft/dts-gen/pull/87 indicates that was just added :-D